testing/camunda-process-test-java/src/test/java/io/camunda/process/test/api/CamundaProcessTestExtensionIT.java

```diff
@@ -56,7 +56,6 @@ void shouldCreateProcessInstance() {
     // then
     CamundaAssert.assertThat(processInstance)
         .isActive()
-        .hasCompletedElements("start")
```

Comment:
🔧 Let's mention affected tests in #26115 and they should be put back along with the fix.

Reply:
Then, it needs to be put back when CPT starts to use v2?